src/main/resources/templates/slurm/command/sinfo

```diff
@@ -0,0 +1 @@
+sinfo -s
```

Comment:
please check `scontrol ping/show` command, maybe it can be more accurate here

Reply:
ping was added
show with entities doesn't display required information. tried to use different entities. show daemon displays only daemons' names
![image](https://user-images.githubusercontent.com/80297657/159476976-0864dd8d-bf02-4677-97ad-8cfed0c0bb58.png)
other helpful commands to get daemon start time wasn't found
start time was filled with minimum date